lib/main.dart

```diff
@@ -1,4 +1,5 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_local_notifications/flutter_local_notifications.dart';
```

Comment:
why did you install another notification package?
There's already one installed by @mayor04 , reach out to him for more